ci-operator/step-registry/workers-rhel/aws-provision/workers-rhel-aws-provision-commands.sh

```diff
@@ -26,6 +31,18 @@ if ! whoami &> /dev/null; then
     fi
 fi
 
+#Get bastion info
+if [[ "${USE_INTERNAL_SSH_BASTION}" == "yes" ]]; then
```

Comment:
Seem like a bastion is a MUST for rhel scaleup. 

So maybe we can drop `USE_INTERNAL_SSH_BASTION` here, and do some pre-check for bastion.
```
if [ -f "${SHARED_DIR}/bastion_public_address" ]; then
  BASTION_SSH_USER=$(cat "${SHARED_DIR}/bastion_ssh_user")
  BASTION_PUBLIC_ADDRESS=$(cat "${SHARED_DIR}/bastion_public_address")
else
  BASTION_SSH_USER="core"
  BASTION_PUBLIC_ADDRESS=$(oc get service ssh-bastion --namespace=test-ssh-bastion --output=jsonpath='{.status.loadBalancer.ingress[0].hostname}')
if 

if [ -z "${BASTION_PUBLIC_ADDRES}" ]; then
  echo "Did not find bastion public address"
  exit 1
else
  export BASTION_SSH_USER
  export BASTION_PUBLIC_ADDRESS
fi
```

Reply:
good point, will update it.